### DISCUSSION TITLE
Active Power vs. Input Power

### DISCUSSION THREAD

**Discussion by Sammy990** [wlcrs/huawei_solar#314] [General]
Hi everyone,

I was not able to find any information, or description of what exactly the Active Power and Input Power from the inverter represent?
What is the value being currently produced by the PV? And what is the other for?

Both seem to be similar with a slight difference in some watts.

Thanks

**Comment by Sammy990** [comment]
Can be closed. I found the description

**Reply by mneundorfer** [reply]
Would you mind sharing it?

**Comment by sportopeden** [comment]
Input power = DC from solarpanels to inverter.
Active power = AC from inverter to house/grid.

**Reply by jires** [reply]
More precise explanation, specially if battery is present in the system:  https://github.com/wlcrs/huawei_solar/discussions/312#discussioncomment-5514424
